phpstan.neon

```diff
@@ -2,6 +2,10 @@ includes:
     - test/phpstan-baseline.neon
 
 parameters:
-    level: 1
+    level: 2
     paths:
         - src
+
+    ignoreErrors:
+        - "#^Constant DEPLOYER_VERSION not found\\.$#"
+        - "#^Constant DEPLOYER_BIN not found\\.$#"
```

Comment:
here we list errors, we dont expect to fix in the long term.

phpstan is not aware that these global constants are defined.
we would need another phpstan bootstrap file to make this clear.